### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Keep config file #86

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -105,7 +102,7 @@ ynh_script_progression --message="Configuring Searx..." --weight=2
 
 secret_key=$(ynh_string_random)
 
-ynh_add_config --template="../conf/settings.yml" --destination="$final_path/searx/settings.yml"
+ynh_add_config --template="../conf/settings.yml" --destination="$final_path/searx/settings.yml" 2>/dev/null
```
Comment (yalh76) [inline scripts/install:105]: why `2>/dev/null` is necessary ?

File: scripts/install [hunk 2]
```diff
@@ -118,7 +115,7 @@ chown -R $app: $final_path
 #=================================================
 ynh_script_progression --message="Configuring uWSGI for Searx..." --weight=2
 
-ynh_add_uwsgi_service
+ynh_add_uwsgi_service 2>/dev/null
```
Comment (yalh76) [inline scripts/install:118]: why `2>/dev/null` is necessary ?

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Comment by ericgaspar** [comment]
!testme

**Review by yalh76 — APPROVED** [review]

**Pull request merged**